src/Components/Board/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from "react";
+import Cell from "../Cell";
+import "./Board.css";
+
+const Board = () => {
+  const [cellsData, setCellsData] = useState(
```

Comment:
Avoid descriptors like `xxxData`, `xxxArray`, `xxxList` in names. Make them evident in `xxx` itself.

`cellsData` could be `boardCells` or even simply `cells`.